Prox/Prox/PlaceDetails/PlaceDetailsCardView.swift

```diff
@@ -242,7 +242,7 @@ class PlaceDetailsCardView: UIView {
         // Labels will gracefully collapse on nil.
         titleLabel.text = place.name
         categoryLabel.text = PlaceUtilities.getString(forCategories: place.categories.names)
-        updateURLText(place.url)
+        updateURLText(place.googleProvider?.url)
```

Comment:
This is a case where we either want to use the google url, or none at all - I don't think we want to show the Yelp url at all, because we don't want to emphasize Yelp more, and there are already 2 other entry points to the Yelp url.